Lib/compression/zstd/zstdfile.py

```diff
@@ -0,0 +1,378 @@
+import builtins
```

Comment:
I am not sure about conventions in CPython, but maybe rename this file to `_zstdfile.py` to convey the fact that `open` and `ZstdFile` should be imported from `compression.zstd` and not from `compression.zstd.zstdfile`?

Alternatively, move the contents into `__init__.py`. After all, this seems what is done e.g. in `lzma.py`.

Reply:
I'm hoping to keep the split of the implementation across files for maintainability purposes. I think it would be fine to rename this file to `_zstdfile.py`, but I don't have strong opinions of if it should be private.

Reply:
We should have one canonical import location/path, so either `from compression.zstd.zstdfile import ZstdFile` or from compression.zstd import ZstdFile`. I am +0 for the latter, and if so I'd suggest making `zstdfile` private as an implementation detail. It's always easier to make it public later than vice-versa.

Reply:
Yes agreed, I think I will make it private.